### DISCUSSION TITLE
[2주차] - 스터디 일정 및 장소 공지 사항

### DISCUSSION THREAD

**Discussion by JoisFe** [Study-2-Effective-Java/Effective-Java#20] [2. 공지 사항]
# 2주차 스터디 일정 및 장소 공지 사항
### 상세 주소 : 서울특별시 강남구 테헤란로2길 32 (강남역 3번출구 도보 2분!)
![image](https://user-images.githubusercontent.com/90208100/208358886-1198a5c2-dc43-448b-a66f-48c7306315cd.png)

### 스터디 룸 결제 내역
![image](https://user-images.githubusercontent.com/90208100/208358989-1a492ee3-342b-487c-acf3-93c6684c9781.png)
* 추가 인원 2인 10,000원은 현장 추가 결제 할 예정

### 참석 인원 : 6인
- [김진완](https://github.com/jinan159)
- [김훈민](https://github.com/bunsung92)
- [나미](https://github.com/gonami-gennnmja)
- [민아영](https://github.com/coalong)
- [유덕린](https://github.com/YuDeokRin)
- [조재철](https://github.com/joisFe)

### 불참 인원 : 2인
- [김도연](https://github.com/chikeem90)
- [이제훈](https://github.com/CoRock)
* 사유 : 개인 사정

**Comment by Irisation23** [comment]
👍 확인 했습니다.

**Comment by YuDeokRin** [comment]
확인 했습니다. 그날 뵙겠습니다.
